jangjuseong/app.js

```diff
@@ -40,6 +35,21 @@ app.get('/ping', (req, res) => {
   res.status(200).json({ message: 'pong' });
 });
 
+app.post('/users/signup', async (req, res, next) => {
+  const { name, email, password, profile_image } = req.body;
+
+  await mysqlDataSource.query(
+    `INSERT INTO
+      users (name, email, password, profile_image)
+      VALUES (?, ?, ?, ?);
+    `,
+    [name, email, password, profile_image]
+  );
+
+  res.status(200).json({ message: 'userCreated' });
+});
```

Comment:
엔드포인트 url을 포함하여 내부 로직 코드를 깔끔하게 잘 작성해주셨네요!

1. 두번째 인수로 받는 콜백 함수의 parameter 중 next 함수는 콜백 함수 내부 로직에서 사용하고 있지 않으니 삭제해주세요!
2. 39번 라인에서 `profile_image` 변수를 구조분해 할당으로 값을 가져오고 있습니다. 다만 JavaScript 언어 고유의 컨벤션 중 하나로 단어 간 띄어쓰기를 해야하는 경우 camelCase를 적용하는 것이 일반적입니다. 언어에 대한 컨벤션도 잘 지키면 가독성과 유지 보수 측면에서도 유리해질 것 같네요!
3. 지금도 충분히 가독성이 좋지만 raw query를 작성하는 부분에서 포맷팅을 조금만 수정해주면 너무 좋을 것  같네요!
4. 함수가 종료되는 부분에는 return 키워드를 포함하여 내가 의도한 코드 이외의 코드가 추가로 실행되지 않도록 작성해주세요!

```suggestion
app.post('/users/signup', async (req, res) => {
  const { name, email, password, profileImage } = req.body;

  await mysqlDataSource.query(
    `INSERT INTO users (
        name,
        email,
        password,
        profile_image
      )
      VALUES (
        ?,
        ?,
        ?,
        ?
      );
    `,
    [name, email, password, profile_image]
  );

  res.status(200).json({ message: 'userCreated' });
});
```

Reply:
상웅님 리뷰 감사합니다~ 
4번 항목의 '함수가 종료되는 부분에는 return 키워드를 포함하여 내가 의도한 코드 이외의 코드가 추가로 실행되지 않도록 작성해주세요!' 에 대한 이해가 잘 가지 않습니다. 추가적으로 설명 부탁드립니다!